acra/configuring-maintaining/logging/_index.md

```diff
@@ -1,4 +1,30 @@
 ---
-title: logging
+
+weight: 1
+title: Logging
 bookCollapseSection: true
 ---
+
+# Logging in Acra
+
+Starting with version [`0.77.0`](https://github.com/cossacklabs/acra/releases/tag/0.77.0), Acra supports three different logging formats: plaintext, [CEF](https://kc.mcafee.com/resources/sites/MCAFEE/content/live/CORP_KNOWLEDGEBASE/78000/KB78712/en_US/CEF_White_Paper_20100722.pdf), JSON.
+
+Acra logs are compatible with various external log analysing tools (like [ELK stack](https://www.elastic.co/elk-stack)) and SIEM systems.
+
+Logging mode and verbosity level can be configured for AcraServer, AcraTranslator and AcraWebConfig in the corresponding `yaml` files or passed as CLI parameter.
+
+Possible values for `logging_format` (case independent): 
+- [plaintext](output/#plaintext)
+- [cef](output/#cef)
+- [json](output/#json)
+
+Feel free to suggest any [logging improvements](improvements/).
+
+Verbose logging can be enabled with -v option on the command line.
```

Comment:
I believe command-line options should be described in a more consistent manner. How about @iamnotacake's style?

```markdown
* `--logging_format`

  Set the logging format to use.

  - `plaintext` (default)
  - `cef`
  - `json`

  The values are case-insensitive.
  [Learn more about supported formats.](formats/)

* `-v`

  Enable verbose logging, useful for debugging.
```